src/Kiota.Builder/Refiners/GoRefiner.cs

```diff
@@ -207,6 +207,7 @@ public override Task Refine(CodeNamespace generatedCode, CancellationToken cance
                 "Error",
                 () => new CodeType { Name = "string", IsNullable = false, IsExternal = true }
             );
+            NormalizeNamespaceNames(generatedCode);
```

Comment:
why do we need to run this twice? it looks like a code smell to me (sides the performance impact)

Reply:
resolved